guides/source/active_record_postgresql.md

```diff
@@ -600,6 +601,23 @@ end
 
 By default `:deferrable` is `false` and the constraint is always checked immediately.
 
+Exclusion Constraints
+---------------------
+
+* [exclusion constraints](https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-EXCLUSION)
+
+```ruby
+# db/migrate/20131220144913_create_products.rb
+create_table :products do |t|
+  t.integer :price, null: false
+  t.daterange :availability_range, null: false
+
+  t.exclusion_constraint "price WITH =, availability_range WITH &&", using: :gist, name: "price_check"
+end
+```
+
+Like foreign keys, exclusion constraints can be deferred by setting `:deferrable` to `:deferred`. By default, `:deferrable` is `:immediate` and the constraint is always checked immediately.
```

Comment:
```suggestion
Like foreign keys, exclusion constraints can be deferred by setting `:deferrable` to `:deferred`. By default, `:deferrable` is `false` and the constraint is always checked immediately.
```

Reply:
The default value of `:deferrable` is `false`

https://github.com/rails/rails/blob/53df1e140d75efaffa63c1b058d8b17834e576a1/activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb#L682

Reply:
Updated. Hopefully the 3rd time is the charm!